certified-connectors/personr/apiProperties.json

```diff
@@ -0,0 +1,21 @@
+{
+  "properties": {
+    "capabilities": [],
```

Comment:
Please update apiProperties.json file as per the below link. 
https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/apiProperties.json